tests/bson/bson-javascript-clone-001.phpt

```diff
@@ -0,0 +1,33 @@
+--TEST--
+MongoDB\BSON\Javascript can be cloned
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/tools.php';
+
+$javascript = new MongoDB\BSON\Javascript("function foo(bar) {var baz = bar; var bar = foo; return bar; }", ['foo' => 42]);
```

Comment:
Is there any reason we might want separate cases for scope and no-scope? I'm fine if you think it's unnecessary.

Reply:
I don't think it is necessary, as the scope logic covers all of the no-scope logic.